### PR TITLE
404 text

### PR DESCRIPTION
**Pull request by mdellabitta**
Updated the text on the 404 page per Jack's request.

### CODE REVIEW

File: src/404.html [hunk 1]
```diff
@@ -87,17 +87,21 @@
 		<cap-nav></cap-nav>
 		<main id="main" class="l-interiorPage">
 			<header class="u-bg-gray-500 u-col-span-full">
-				<cap-page-header heading="Hmm... can't find that.">
+				<cap-page-header heading="Not Found.">
 					<p class="u-text-white u-text-serif">
-						Maybe the last borrower didn't return that link.
+						The Caselaw Access Project has updated and simplified our site to
+						provide long term, unrestricted access to U.S. case law.
```
Comment (bensteinberg) [inline src/404.html:93]:
```suggestion
						provide long-term, unrestricted access to U.S. case law.
```

Comment (mdellabitta) [reply src/404.html:93]: Thank you. I should have tagged you in explicitly!

File: src/404.html [hunk 2]
```diff
@@ -87,17 +87,21 @@
...
+						provide long-term, unrestricted access to U.S. case law.
 					</p>
 					<p class="u-text-white u-text-serif">
-						If you got here through our site or an article we host, please let
-						us know at
+						As part of this process we have deactivated portions of the site,
+						potentially including the page you were looking for.
+					</p>
+					<p class="u-text-white u-text-serif">
+						If you have questions about the changes we have made, please let us
```
Comment (bensteinberg) [inline src/404.html:100]:
```suggestion
						If you have questions about the changes we’ve made, please let us
```
Two "haves" seems odd here.

**Review by bensteinberg — APPROVED** [review]
Consider one additional change, but no pressure.

**Pull request merged**